db/migrate/20200430002042_create_work_experiences.rb

```diff
@@ -0,0 +1,15 @@
+class CreateWorkExperiences < ActiveRecord::Migration[5.2]
+  def change
+    create_table :work_experiences do |t|
+      t.references :student, foreign_key: true, null: false
+      t.references :company, foreign_key: true, null: false
+      t.string :entity
+      t.string :country
+      t.string :title
+      t.string :study_area
+      t.integer :degree_level
```

Comment:
I am aware that in the excel that I2T2 provided us they included degree_level as a field. I think that we can omit it, since we have that information in the student model.